LabApiUtilities/build.gradle

```diff
@@ -79,7 +79,17 @@ afterEvaluate {
     // above in the plugins block.
     compileJava.dependsOn generateBuildConfig, generateTestBuildConfig
 }
+tasks.withType(JavaCompile) {
```

Comment:
Let's add a comment here and in description section on why this change was needed. If you have a link where this change is needed as per AGP 8+, let's add that as well